subgit/core.py

```diff
@@ -486,6 +487,111 @@ def pull(self, names):
                 log.info(f"Current commit summary on HEAD in git repo '{name}': ")
                 log.info(f"  {str(repo.head.commit.summary)}")
 
+    def delete_local_repo(self, repo_names=None):
+        """
+        Method takes zero or one repo name as argument.
+
+        Checks if the repo(s) are valid git repo(s) and if there are any untracked changes.
+        """
+
+        config = self._get_config_file()
+
+        active_repos = self._get_active_repos(config)
+
+        if repo_names is None:  # If no names are specified in the command
+            active_repos_dict = config.get('repos', [])
+
+            repo_choices = ", ".join(active_repos_dict)
+
+            answer = self.yes_no(f"Are you sure you want to delete the following repos '{repo_choices}'")
+
+            if answer:
+                self.repo_deletion(active_repos=active_repos)
+
+        if isinstance(repo_names, list):  # If at least one name was specified
+
+            repo_choices = ", ".join(repo_names)
+
+            answer = self.yes_no(f"Are you sure you want to delete the following repos '{repo_choices}'")
+
+            if answer:
+                self.repo_deletion(active_repos, repo_names)
+
+    def repo_deletion(self, active_repos, repos=None):
+        """
+        Helper method that recieves a list of repos to delete and takes care of the logic
+        """
+        if not repos:
+            repos = active_repos
+
+        in_cnf_file = True
+
+        dirty_repos = False
+
+        repo_paths = []
+
+        path_no_exist = []
+
+        good_repos = []
+
+        for name in repos:
+            repo_paths.append(os.path.join(os.getcwd(), name))
+            if name not in active_repos:
+                log.warning(f"Argument: '{name}' does not exist in '.subgit.yml'")
+                in_cnf_file = False
+
+        if not in_cnf_file:
+            return 1
+
+        for path in repo_paths:
+
+            if not os.path.exists(path):
+                log.warning(f'Path to repo does not exist: {os.path.basename(path)} | Skipping {os.path.basename(path)}')
```

Comment:
You are mixing which one you use for strings, we usually stick with `" "` for the string and `' '` inside the strings. You keep mixing them so they should be streamlined to the same one